WARNING:TorrentChecker:no trackers, skip GUI request.

While running latest Ubuntu-64 build. Complete screen is filled with a tracker error. Some hints in error log, _WARNING:TorrentChecker:no trackers, skip GUI request._ no TraceBack.

![tribler_7_error__no_tracker_available](https://cloud.githubusercontent.com/assets/325224/20315353/d9b48834-ab5d-11e6-88d0-3d9f6bd614da.png)

```
WARNING:Dispersy:received identical message channel 666@6 from <Candidate 111.220.161.233:23514>
WARNING:PreviewChannelCommunity:drop a 132 byte packet duplicate message by global_time (2) [channel] from <Candidate 111.220.161.233:23514>
WARNING:TunnelLogger:Got create (1763782181) from ('109.189.179.82', 45335), I am <Member 6 78e7f393e3256a792f96ca5a841cb0115740b5ba>
WARNING:TunnelLogger:Got extend (1763782181) from ('109.189.179.82', 45335), I am <Member 6 78e7f393e3256a792f96ca5a841cb0115740b5ba>
WARNING:TunnelLogger:Got created (281040662) from ('95.211.155.142', 7552), I am <Member 6 78e7f393e3256a792f96ca5a841cb0115740b5ba>
WARNING:TorrentChecker:no trackers, skip GUI request. infohash: fc870ed139dd26e51ba5f1e117212521463f527c
WARNING:TorrentChecker:no trackers, skip GUI request. infohash: 420ffa5cb90241d398a75fa6ab314b4d7b7e1eac
WARNING:TorrentChecker:no trackers, skip GUI request. infohash: f9c96192632079dd0a25ed75975d470a6c3390aa
WARNING:TorrentChecker:no trackers, skip GUI request. infohash: f8dc563728f03660f56819bd2c0b30a4e69c5c92
WARNING:TorrentChecker:no trackers, skip GUI request. infohash: fa07327885e58f0ba93c4378d3a53d9de6b8a602
WARNING:Dispersy:received identical message channel 14372@6 from <Candidate 77.251.80.151:7759>
WARNING:PreviewChannelCommunity:drop a 224 byte packet duplicate message by global_time (2) [channel] from <Candidate 77.251.80.151:7759>
WARNING:Dispersy:received identical message channel 664@6 from <Candidate 77.251.80.151:7759>
WARNING:PreviewChannelCommunity:drop a 312 byte packet duplicate message by global_time (2) [channel] from <Candidate 77.251.80.151:7759>
WARNING:Dispersy:received identical message channel 5265@6 from <Candidate 77.251.80.151:7759>
```

More info appeared after a close ```UDP tracker failed for url udp://tracker.novalayer.org:6969```:
```
ERROR:root:Traceback (most recent call last):
  File "/usr/share/tribler/TriblerGUI/event_request_manager.py", line 75, in on_read_data
    raise RuntimeError(json_dict["event"]["text"])
RuntimeError: [Failure instance: Traceback (failure with no frames): <type 'exceptions.ValueError'>: UDP tracker failed for url udp://tracker.novalayer.org:6969
]

Unhandled Error
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/share/tribler/TriblerGUI/core_manager.py", line 53, in start_tribler_core
    reactor.run()
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 1194, in run
    self.mainLoop()
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 1203, in mainLoop
    self.runUntilCurrent()
--- <exception caught here> ---
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 798, in runUntilCurrent
    f(*a, **kw)
  File "/usr/share/tribler/Tribler/community/multichain/community.py", line 411, in on_tunnel_remove
    self.schedule_block(candidate, tunnel.bytes_up, tunnel.bytes_down)
  File "/usr/share/tribler/Tribler/community/multichain/community.py", line 144, in schedule_block
    self.publish_signature_request_message(candidate, total_amount_sent_mb, total_amount_received_mb)
  File "/usr/share/tribler/Tribler/community/multichain/community.py", line 158, in publish_signature_request_message
    message = self.create_signature_request_message(candidate, up, down)
  File "/usr/share/tribler/Tribler/community/multichain/community.py", line 172, in create_signature_request_message
    total_up_requester, total_down_requester = self._get_next_total(up, down)
  File "/usr/share/tribler/Tribler/community/multichain/community.py", line 378, in _get_next_total
    total_up, total_down = self.persistence.get_total(self._public_key)
  File "/usr/share/tribler/Tribler/community/multichain/database.py", line 273, in get_total
    db_result = self.execute(db_query, (public_key, public_key)).fetchone()
  File "/usr/share/tribler/Tribler/dispersy/util.py", line 150, in wrapper
    return_value = func(*args, **kargs)
  File "/usr/share/tribler/Tribler/dispersy/database.py", line 277, in execute
    assert self._cursor is not None, "Database.close() has been called or Database.open() has not been called"
exceptions.AssertionError: Database.close() has been called or Database.open() has not been called

Stopping Tribler...
```

--- a/Tribler/Core/TorrentChecker/session.py
+++ b/Tribler/Core/TorrentChecker/session.py
@@ -475,7 +475,9 @@ class UdpTrackerSession(TrackerSession):
         in the session has failed and thus no data can be obtained.
         """
         self._is_failed = True
-        self.scraper.stop()
+        if self.scraper:
+            self.scraper.stop()
+
         if self.result_deferred:
             result_msg = "UDP tracker failed for url %s" % self._tracker_url
             if msg:

--- a/Tribler/Test/API/test_download.py
+++ b/Tribler/Test/API/test_download.py
@@ -2,6 +2,7 @@ from binascii import hexlify
 import logging
 import os
 import shutil
+from unittest import skip
 from twisted.internet.defer import Deferred
 from Tribler.Core.Utilities.network_utils import get_random_port
 from Tribler.Core.Utilities.twisted_thread import deferred
@@ -45,6 +46,7 @@ class TestDownload(TestAsServer):
         d.addCallback(self.on_download)
         return self.test_deferred
 
+    @skip("Fetching a torrent from the external network is unreliable")
     @deferred(timeout=60)
     def test_download_torrent_from_magnet(self):
         magnet_link = 'magnet:?xt=urn:btih:%s' % hexlify(UBUNTU_1504_INFOHASH)

